hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/Service.java

```diff
@@ -28,6 +29,7 @@
  * crypto-service, token-service etc.,
  */
 public interface Service {
+    SemanticVersion CURRENT_VERSION = new SemanticVersion(0, 45, 0, "SNAPSHOT", "");
```

Comment:
We're not permanently using this version, are we? The testnet and mainnet migrations will need to do a genesis initialization

Reply:
Why will testnet and mainnet do genesis migration ?

Reply:
There _may_ be a Schema for each version, but it's not required. The thing about the schemas is _each schema assumes that the migrations for all previous schemas have already been done_

Reply:
yes, but my question is should we keep all the schemas for every migration we add in repo.